tests/plotting/test_widgets.py

```diff
@@ -609,3 +609,94 @@ def test_logo_widget(verify_image_cache):
     pl = pv.Plotter()
     with pytest.raises(TypeError, match='must be a pyvista.ImageData or a file path'):
         pl.add_logo_widget(logo=0)
+
+
+def test_clear_box_widget(verify_image_cache):
+    mesh = examples.download_doorman()
```

Comment:
Instead of the doorman example, would you please use a simple geometry like a `pv.Cube`? My thoughts being:

1. The doorman has `29349` points and `9783` cells while we want tests to be as lightweight as possible. Since this test is not dependent on the data being rendered, let's make sure that data is as small as possible.
2. Simple geometry will help the image regression validator... that thing isn't perfect and can pick up on all kinds of subtle differences. We want to pick up on the presence of the widgets in the scene here, so let's make sure the rest of the scene is a simple as possible.

Reply:
Though, maybe this point is moot and the performance difference is negligible? 🤷🏻

Reply:
Every little bit counts. Let's change it.